devTools/svgTester/export-graphs.ts

```diff
@@ -6,17 +6,26 @@ import * as fs from "fs-extra"
 
 import * as path from "path"
 import workerpool from "workerpool"
+
+function parseArgAsList(arg?: string): string[] {
+    return (arg ?? "")
+        .toString()
+        .split(",")
+        .map(String)
+        .filter((entry: string) => entry)
```

Comment:
nit: don't need the `toString()`, and also not the `map(String)`, since these are all strings anyhow.

Reply:
Oh, I realised that my types were off here! If you pass a single config id with `-c 50`, then minimist will parse that as a number. So, I kept `toString()` but changed the type of the incoming arg to `unknown`. And I removed `map(String) – that really is unnecessary! Thanks :)